quickOrder/Inventory.py

```diff
@@ -1,4 +1,4 @@
-class Inventory:
+class InventoryItem:
     def __init__(self, item_id, item_name, current_quantity, min_quantity, isOrdered, status, refillNeeded):
         self.item_id = item_id
         self.item_name = item_name
```

Comment:
Are you following the camel case or snake case in variables? Please stick to one.